discovery-provider/src/api/v1/playlists.py

```diff
@@ -120,7 +115,13 @@ class Playlist(Resource):
     @cache(ttl_sec=5)
     def get(self, playlist_id):
         playlist_id = decode_with_abort(playlist_id, ns)
-        playlist = get_playlist(playlist_id, None)
+        playlist = get_playlist(
+            {
+                "with_users": True,
+                "current_user_id": None,
+                "playlist_ids": [playlist_id],
```

Comment:
why do we need both playlist_id and playlist_ids?

Reply:
sorry this was my bad will update